Lesson4/src/service/MilitaryOffice.java

```diff
@@ -1,23 +1,23 @@
 package service;
 
+import model.Address;
 import model.Person;
 
 import java.util.List;
 
 public class MilitaryOffice {
     private PersonRegistry personRegistry;
-    List<Person> registry;
 
     public MilitaryOffice(PersonRegistry personRegistry) {
         this.personRegistry = personRegistry;
-        registry = personRegistry.getCitizens();
     }
 
-    public String suitable() {
+    public String suitable(Address address) {
         StringBuilder names = new StringBuilder();
         System.out.println("who is fit for the army: ");
-        for (Person person : registry) {
-            if (person.getAddress().getCountry().equals("Belarus") &&
+        for (Person person : personRegistry.getCitizens()) {
```

Comment:
call getPeople instead of getCitizens and you won't need to filter by address because you will get already filtered result. It's not responsibility of MilitaryOffice to filter people by address, the just need to filter by age and sex

Reply:
if I call getPeople() why do I need to give Address to suitable()? Isn't that the same thing?

Reply:
the idea is to make each class do it's work

People registry works with citizens and can give you list of citizens by address

MilitaryOffice works with the registry to get list of all people by given address and then to figure out who is fit for military service. 

Figuring out if a person is living by given address or not - it's not responsibility of the military office, it's responsibility of registry